src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -499,6 +499,7 @@ object TezosTypes {
       blockLevel: Int,
       timestamp: Option[Instant],
       cycle: Option[Int],
+      period: Option[Int],
```

Comment:
Just wondering: are we putting too much inside this "tagging" object? Should we consider a better approach?

Reply:
Maybe at some point when reading data from checkpoints we could just read whole block and literally `tagWithBlock` not only with the small part of it. But at this point I don't think that redesign is necessary